### ISSUE TITLE
LABELS_TO_MEASURE causes the action to fail

### ISSUE THREAD

**Issue by cmayor** [github/issue-metrics#180]
If LABELS_TO_MEASURE is set then the action will fail with:

```
Starting issue-metrics search...
  File "/action/workspace/issue_metrics.py", line 331, in <module>
    main()
  File "/action/workspace/issue_metrics.py", line 262, in main
    labels = labels.split(",")
             ^^^^^^^^^^^^
AttributeError: 'list' object has no attribute 'split'
```
It looks like the block [here](https://github.com/github/issue-metrics/blob/main/issue_metrics.py#L261) is redundant after [this PR](https://github.com/github/issue-metrics/pull/171) which already converts the environment string to a list.

**Comment by zkoppert** [comment]
Thanks for submitting this issue! I'm currently on holiday so I'll take a look at this when I return in January.

**Comment by hianhianhian** [comment]
Hi, I'm currently blocked by this issue so I figured I could just use an older version of the action (e.g. v2.14.0), so I changed my workflow yml file as follows
```yml
- name: Run issue-metrics tool
   uses: github/issue-metrics@v2.14.0
```

However, I still run into this issue. It seems that even if I use an older version of the action, it's still pulling the latest Docker image ie. https://github.com/github/issue-metrics/blob/ebc6a99b31b69b73a26678b9ac5a6ebbc021c663/action.yml#L7

I'm not too familiar with Docker, but was just wondering if anything can be done to workaround this?
